cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -680,7 +680,7 @@ func (command *Evaluate) evaluateKubernetes(ctx *evaluate.Context) (err error) {
 		}
 
 		// Commands for the evaluation to run inside the container.
-		resultPath := "/var/evaluation/" + log.CleanModelNameForFileSystem(model.ID())
+		resultPath := "/var/evaluations/" + log.CleanModelNameForFileSystem(model.ID())
```

Comment:
guess we have no test for kubernetes...

Reply:
Nope

Reply:
Can we add a note in the issues that we are missing system tests in the kubernetes runtime?

Reply:
It is currently not possible to have system tests for kubernetes.